src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.cs

```diff
@@ -472,14 +472,29 @@ private void QueueApplyReplacements()
             }
 
             var asyncToken = _asyncListener.BeginAsyncOperation(nameof(QueueApplyReplacements));
-            _conflictResolutionTask.SafeContinueWith(
-                t => ApplyReplacements(t.Result, _conflictResolutionTaskCancellationSource.Token),
-                _conflictResolutionTaskCancellationSource.Token,
-                TaskContinuationOptions.OnlyOnRanToCompletion,
-                ForegroundTaskScheduler).CompletesAsyncOperation(asyncToken);
+            _conflictResolutionTask
```

Comment:
I'll leave this for a future update to this code. The style seen in this PR matches a great deal of code seen through this file.